### PR TITLE
Data cleaning

### PR DESCRIPTION
**Pull request by Drishtantr**
JS Diff algorithm implemented, made data dynamic, added an original transcription of an audio file which later is compared to the user input and difference is calculated. For now, the implemented parameters are Accuracy, Grammar, incorrect words from the total input words, suggestions for incorrect grammar.

### CODE REVIEW

File: src/views/Result.vue [hunk 1]
```diff
@@ -10,8 +10,8 @@
         <ul class="list-group text-left m-2">
           <li class="list-group-item">Rank: 6</li>
           <li class="list-group-item">Time:{{timeTaken}}</li>
-          <li class="list-group-item">Accuracy: {{grammerAccuracy}}</li>
-          <li class="list-group-item">Strongest: Grammar</li>
+          <li class="list-group-item">Accuracy: {{jsDiff}}%</li>
+          <li class="list-group-item">Grammar: {{grammerAccuracy}}%</li>
```
Comment (shakyaprasen) [inline src/views/Result.vue:14]: still wrong spelling of grammar

File: src/views/Result.vue [hunk 2]
```diff
@@ -163,10 +167,26 @@ export default {
         this.acceptAlert('danger', 'Error', 'Error Occured during downloading.');
       }
     },
+    quoteText() {
+      if (this.jsDiff > 89) {
```
Comment (shakyaprasen) [inline src/views/Result.vue:171]: these sentences can be moved to a const file and exported from there

**Review by shakyaprasen — APPROVED** [review]

File: src/views/Result.vue (hunk 1 above)
Comment (Drishtantr) [reply src/views/Result.vue]: corrected

File: src/views/Result.vue (hunk 2 above)
Comment (Drishtantr) [reply src/views/Result.vue]: done

**Pull request merged**